Components/CategoryResolver.php

```diff
@@ -90,8 +90,8 @@ public function storeRemoteCategories(array $categories, $articleId)
 
         $this->manager->flush();
 
-        $this->addProductToRemoteCategory($remoteCategories, $articleId);
         $this->removeProductsFromRemoteCategory($remoteCategories, $articleId);
+        $this->addProductToRemoteCategory($remoteCategories, $articleId);
```

Comment:
what is the benefit to call ```addProductToRemoteCategory``` after ```addProductToRemoteCategory```?
Less iterations in removeProductsFromRemoteCategory?

Reply:
exactly
actual less iterations in remove and in add